source/sonata_tech.rst

```diff
@@ -358,9 +358,9 @@ This type of connectivity happens between astrocytes. The properties are similar
 ``source_node_id`` and ``target_node_id`` datasets have an HDF5 attribute of type string named ``node_population`` defining the source and target node population name respectively.
 
 
-Fields for electrical_synapse connection type edges
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Connection type is ``electrical_synapse``. Used for gap junctions between neurons
+Fields for electrical connection type edges
```

Comment:
I swear we've had this discussion before.
Regardless, we need to look at what would need to be changed, probably based on:
https://bbpteam.epfl.ch/source/search?full=electrical_synapse&defs=&refs=&path=&hist=&type=&xrd=&nn=16&si=full&searchall=true&si=full

Reply:
We have discussed it on several occasions:
* https://github.com/BlueBrain/libsonata/issues/229
  * > "electrical" is the correct type; so let's use that 
* https://bbpgitlab.epfl.ch/common/doc/circuit-documentation/-/merge_requests/10#note_68006
  * > I think we went for `chemical` bc we wanted to put astrocyte-vasculature connection there too but later ended up creating `endfoot` connection type instead. As the reference circuit is already with `chemical` and `electrical`, the best is probably to move the examples and any other reference to that pattern.
  * > ok, then let's use `chemical` and `electrical`. 

Just for reference, here are the original discussions of the naming (gerrit archives)
* https://bbpgitlab.epfl.ch/nse/archive/gerrit-reviews/-/blob/main/common/circuit-documentation/45716-Add_initial_SONATA_format_requirements-Ia2d0f13381bdef05c4972c07588934a71c0df8c4.md
* https://bbpgitlab.epfl.ch/nse/archive/gerrit-reviews/-/blob/main/common/circuit-documentation/45864-Getting_Dans_feedback-Ic360b469bff32957662d81fa17dee34c3d8f7490.md

It just haven't been updated everywhere. I have already changed it in some places:
* [commit](https://bbpgitlab.epfl.ch/nse/sonata-generator/-/commit/571f380e83bf72a6574ebc5a4a364c07a48f6a1c) changing it in sonata-generator
* snap only allows `electrical` in the [schemas](https://github.com/BlueBrain/snap/tree/master/bluepysnap/schemas/edge)

Reply:
my bad, yes we should go for `electrical`

Reply:
@jdcourcol, @mgeplf : is this good to go?